src/common/SPELLS/talents/evoker.ts

```diff
@@ -0,0 +1,9 @@
+// Generated file, changes will be overwritten!
+import { SpellList } from '../Spell';
+
+const talents: SpellList = {
+  //Shared
+  //Devastation
+  //Feral
```

Comment:
feral?

Reply:
Ah yes! i compied the balance file. I updated this in another ready to go PR that is based on this where it has been renamed. But Looking over the changes @Darthelit made on the file i think its best to accept their PR on the SpellList.